reemgawad/brunchtime
# Fixing seeds

What I Changed:
- fixed seed infomration
-  added new restaurants
-  changed profile pic + name to philippine

How to check:
- Go to browser and check each individual restaurant 
- go to navbar and check pic + name
